### DISCUSSION TITLE
[Feature] [Mobile] Search for albums and faces

### DISCUSSION THREAD

**Discussion by Hely0n** [immich-app/immich#8170] [Feature Request]
### The feature

If you have a huge amount of faces and albums, it's very tedious to show the photos "of that one birthday party we talked about" at a party/gathering, as you have to scroll and search manually in the app. 

The same applies to faces, when you want to show all photos/videos with someone specific.

The web implementation is perfect, but it really lacks on mobile. It would be very nice and helpful, if this feature will be added 😊

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by frederikb96** [comment]
I am also truly interested in this feature, have multiple hundred albums :)
At the same time, it would be nice to not only have a search possibility for own albums but also shared once, since I have the same problem there.

The part with searching for albums is also asked in #8240 :)

**Comment by bo0tzz** [comment]
This is now implemented

**Reply by Hely0n** [reply]
Do you mean the album search or the face search too? I think the face search is still missing.

**Reply by bo0tzz** [reply]
The search tab has a bunch of filter chips at the top, including people.

**Reply by Hely0n** [reply]
Yes, but you again have to find the person without a search. I have like 300 faces (was a huge amount of work, I know 😂) and whenever I'm at a family event for example and want to show an uncle some photos from the past with him, I can't easily find him, as the faces are not even sorted alphabetically. That's why a face search would be nice in the face selection, aswell as the person filter selection, just like on web.

**Reply by frederikb96** [reply]
Yes that's true, even in search tab, you cannot search for people. It is just a filter where you can pick a person from the list 🙂
Having the option in search mode would probably be enough 🙂
